MarkerMetro.Unity.WinLegacyUnity/System/Security/Cryptography/EncryptionProvider.cs

```diff
@@ -137,11 +137,10 @@ public static string Decrypt(string cipherString, string key)
 
                 return strDecrypted;
             }
-            catch (Exception)
+            catch (Exception ex)
             {
-                // MetroEventSource.Log.Error(ex.Message);
-                //throw;
-                return "";
+               System.Diagnostics.Debug.WriteLine(ex.Message);
+               throw new Exception("[EncryptionProvider] Error Decrypting a string");
```

Comment:
You are changing from failing silently (returning an empty string) to throwing an exception. I'm not a fan of failing silence but the person that did it must have done it for a reason... Worth seeing the history and checking why this was happening.